flink-runtime/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcActorTest.java

```diff
@@ -515,6 +542,29 @@ public void setFoobar(int value) {
 
 	// ------------------------------------------------------------------------
 
+	interface NullRespondingGateway extends DummyRpcGateway {
```

Comment:
Why do we introduce new classes/interfaces here? Can't we just use the already existing `DummyRpcGateway`/`NullRespondingEndpoint`?

Reply:
Would also be possible but I didn't want to implement `Integer synchronousFoobar()` for the `DummyRpcEndpoint`. Hence, I opted for a new interface.